functions/__fzf_preview_file.fish

```diff
@@ -23,6 +23,6 @@ function __fzf_preview_file --argument-names file_path --description "Print a pr
     else if test -p "$file_path"
         __fzf_report_file_type "$file_path" "named pipe"
     else
-        echo "Unknown file type." >&2
+        echo "File doesn't exist." >&2
```

Comment:
Fixed a small bug: realized the only time this happens is if the file doesn't exist.